libs/ui-lib/lib/ocm/components/clusterConfiguration/networkConfiguration/NetworkConfiguration.tsx

```diff
@@ -223,19 +223,6 @@ const NetworkConfiguration = ({
         />
       )}
 
-      {!isUserManagedNetworking &&
-        !featureSupportLevelData.isFeatureSupported('CLUSTER_MANAGED_NETWORKING_WITH_VMS') && (
-          <Alert
-            title="Your cluster will be subject to support limitations"
-            variant={AlertVariant.info}
-            isInline={true}
-            data-testid="networking-vms-alert"
-          >
-            Some or all of your discovered hosts are virtual machines, so selecting the
-            cluster-managed networking option will limit your installed cluster's support.
-          </Alert>
-        )}
-
```

Comment:
Why are you removing this?

Reply:
This feature don't exists more: CLUSTER_MANAGED_NETWORKING_WITH_VMS

Reply:
We need to double check w/QE, I think @talhil-rh added a test in order to trigger this message...